client/src/Overview/components/StyleSelector.jsx

```diff
@@ -2,21 +2,41 @@ import React from 'react';
 
 const StyleSelector = (props) => {
   if (props.state.styles.length > 0) {
-    let key = -1;
+    let keyOne = -1;
+    let keyTwo = -1;
 
   return (
-    <div className = "styleSelector">
+    <div className = "style-selector">
       <p>STYLE > {props.state.styles[props.OverviewState.styleIndex].name}</p>
       <table>
         <tbody>
         <tr>
            {props.state.styles.map((style)=> {
-             key += 1;
-          return (
-            <td key = {key} onClick = {props.changeStyle}>
-            <img value = {key} src={style.photos[0].thumbnail_url} width="50" height="50"></img>
+             keyOne += 1;
+             if (keyOne >= 4) {
+               return;
+             } else {
+               return (
+                 <td key = {keyOne} onClick = {props.changeStyle}>
+            <img value = {keyOne} src={style.photos[0].thumbnail_url} className = "style-selector-image"></img>
             </td>
             )
+          }
+        })
+       }
+      </tr>
+      <tr>
+           {props.state.styles.map((style)=> {
+             keyTwo += 1;
+             if (keyTwo < 4) {
```

Comment:
I don't totally understand what you are doing here and on line 16 in this file. Is this how you are managing the number of images being displayed?

Reply:
Hey man thanks for looking at the code. I'm trying to get them into rows of 4 images. This is definateley not the best way to do it but it was taking to long to figure out a better way.

Reply:
this will be refactored in later commits